presetup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+sudo apt update
+sudo apt install apt-transport-https ca-certificates curl gnupg2 software-properties-common
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+sudo apt update
+sudo apt-cache policy docker-ce
+sudo apt-get install \
+  git \
+  docker-ce \
+  docker-ce-cli \
+  containerd.io \
+  docker-buildx-plugin \
+  docker-compose \
```

Comment:
Remove docker-compose, the plugin is better!

Reply:
But when I had only the plugin the `docker-compose` command did not work. Maybe I did something wrong. In the tutorial I used they also use only the plugin.

Reply:
The command is then 'docker compose'
It's a subcommand of Docker then.